### DISCUSSION TITLE
Incomplete Industry session status

### DISCUSSION THREAD

**Discussion by MuskanAgarwal123** [github/india#209] [Students]
Hi, even after watching all the videos thrice at one go, it showing incomplete. Wrote a mail that was also unanswered. Look after this as its consuming too much time and lot of our productive hours is getting wasted in watching the same thing thrice. This is not only faced by me but many others too.

**Comment by ankitkushawaha1000** [comment]
i also faced this issue

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
